src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.nuspec

```diff
@@ -7,13 +7,7 @@
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <projectUrl>https://github.com/Microsoft/perfview</projectUrl>
-    <licenseUrl>https://github.com/Microsoft/perfview/blob/master/LICENSE.TXT</licenseUrl>
```

Comment:
I know that licenseurl has been deprecated, but I think we still need to have the license in the package.  What's the right way to do that?

Reply:
It's covered here, in the steps for "Manually edited nuspec file":
https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128#solution

Reply:
According to https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file, it's either/or. This suggests it is not expected that a package would need both.

Reply:
Oh, I see.  Since it's MIT, license type should work.